caffe2/core/context_gpu.h

```diff
@@ -278,6 +278,7 @@ class CAFFE2_CUDA_API CUDAContext final : public BaseContext {
   }
 
   at::Device device() const override {
+    // cache?
```

Comment:
What do you mean by 'cache?'

Reply:
Oh, I'm considering instead of construct a at::Device every time, should we cache the result